objects/client.c

```diff
@@ -2834,25 +2779,6 @@ luaA_client_get_icon(lua_State *L, client_t *c)
 }
 
 static int
-luaA_client_get_focusable(lua_State *L, client_t *c)
-{
-    bool ret;
-
-    if (c->focusable_set)
-        ret = c->focusable;
-
-    /* A client can be focused if it doesnt have the "nofocus" hint...*/
-    else if (!c->nofocus)
-        ret = true;
-    else
-        /* ...or if it knows the WM_TAKE_FOCUS protocol */
-        ret = client_hasproto(c, WM_TAKE_FOCUS);
```

Comment:
This commit loses the above defaults. Also, you cannot check `client_hasproto(c, WM_TAKE_FOCUS)` from Lua and thus this cannot be moved into Lua. (we could export `WM_PROTOCOLS` to Lua, but for that we'd need a good use case first; I don't think this one is one)